security/cmd/node_agent_k8s/main.go

```diff
@@ -409,6 +418,8 @@ func main() {
 		"Vault sign CSR path")
 	rootCmd.PersistentFlags().StringVar(&serverOptions.VaultTLSRootCert, vaultTLSRootCertFlag, "",
 		"Vault TLS root certificate")
+	rootCmd.PersistentFlags().BoolVar(&serverOptions.Pkcs8Keys, pkcs8KeysFlag,
```

Comment:
Can we only use the ENV variable, not the flag?

Reply:
Done. Thanks.